src/Components/Search/NewSearch/NewSearchBar.tsx

```diff
@@ -3,19 +3,23 @@ import { NewSearchBarInputQueryRenderer } from "Components/Search/NewSearch/NewS
 import { Media } from "Utils/Responsive"
 import { MobileSearchBarQueryRenderer } from "./Mobile/MobileSearchBar"
 import { FC } from "react"
+import { getSearchTerm } from "Components/Search/NewSearch/utils/getSearchTerm"
+import { isServer } from "Server/isServer"
 
 interface NewSearchBarProps {
   onClose: () => void
 }
 
 export const NewSearchBar: FC<NewSearchBarProps> = ({ onClose }) => {
+  const urlSearchTerm = isServer ? undefined : getSearchTerm(window?.location)
```

Comment:
This change is not related to the PR's title. It fixes the case when user navigates to the `search?term=something` page and click on the search input. Search should be already pre-populated at this point.

https://github.com/artsy/force/assets/3934579/ac1b48f0-f259-406e-afc3-279b44533b03

Reply:
I really need to change the name of `isServer` to: `__DO_NOT_USE_OUTSIDE_OF_EFFECT_isServer__` or something. You can't call this in a render function; will cause a mismatch

Reply:
Or better, I guess I can wrap this in a hook.